Example/Tests/Network/AlchemyNetworkTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// This source file is part of the Web3Swift.io open source project
+// Copyright 2019 The Web3Swift Authors
+// Licensed under Apache License v2.0
+//
+// AlchemyNetworkTests.swift
+//
+// Created by Vadim Koleoshkin on 27/03/2019
+//
+
+import Nimble
+import Quick
+@testable import Web3Swift
+import XCTest
+
+class AlchemyNetworkTests: XCTestCase {
```

Comment:
It is not really about the responsibility of this object since if it returns network ID then Alchemy servers are running. I am working on ERC20 implementation and will add such tests in that library. For Alchemy I will add checking of Ethereum balance at some address.

Reply:
Ok